lib/validation.js

```diff
@@ -18,10 +18,12 @@
  *
  * @module numberValidation
  */
+const isDecimal = require('validator/lib/isDecimal');
+const isInt = require('validator/lib/isInt');
 
-module.exports = function numberValidation(input, settings) {
-  if (input.target.value === '' && !settings.target.empty) {
-    return `${input.target.name} cannot be left blank!`;
+module.exports = function numberValidation(input) {
+  if (input.target.value && !isInt(input.target.value.toString()) && !isDecimal(input.target.value.toString())) {
```

Comment:
actually, this is correct. It checks if there 

1. is a value
2. if that value is _not_ an integer
3. and if that value is _not_ a decimal

if it is `true` for either the integer or or decimal, then this conditional is false.

the positive check alternative:
```
  if (input.target.value && !(isInt(input.target.value.toString()) || isDecimal(input.target.value.toString()))) {
```
does work as well.

1. has a value
2. if that value is an integer
3. or if that value is a decimal

So, it sounds like you want `or` (112 characters) instead of `and` (111 characters.)